src/internal.c

```diff
@@ -1764,12 +1764,13 @@ int SetSSL_CTX(WOLFSSL* ssl, WOLFSSL_CTX* ctx)
     byte havePSK = 0;
     byte haveAnon = 0;
     byte haveRSA = 0;
-    byte newSSL = ssl->ctx == NULL;
     (void) haveAnon; /* Squash unused var warnings */
 
     if(!ssl || !ctx || ssl->suites == NULL)
         return BAD_FUNC_ARG;
 
+    byte newSSL = ssl->ctx == NULL; /* Assign after null check */
+
```

Comment:
Hey Kaleb, please move the declaration back up to where it was and just do the assignment here. To remain C89 compliant it requires all declarations to be at top of function or in {}. In this case top is correct.